testbenches/simlibs.mk

```diff
@@ -116,9 +116,9 @@ VSOURCES += $(BSG_MANYCORE_DIR)/testbenches/common/v/infinite_mem_profiler.v
 
 VSOURCES += $(BSG_MANYCORE_DIR)/v/bsg_manycore_link_sif_async_buffer.v
 
-VSOURCES += $(BASEJUMP_STL_DIR)/bsg_async/bsg_async_fifo.v
-VSOURCES += $(BASEJUMP_STL_DIR)/bsg_async/bsg_async_ptr_gray.v
-VSOURCES += $(BASEJUMP_STL_DIR)/bsg_async/bsg_launch_sync_sync.v
+#VSOURCES += $(BASEJUMP_STL_DIR)/bsg_async/bsg_async_fifo.v
+#VSOURCES += $(BASEJUMP_STL_DIR)/bsg_async/bsg_async_ptr_gray.v
+#VSOURCES += $(BASEJUMP_STL_DIR)/bsg_async/bsg_launch_sync_sync.v
```

Comment:
Are these no longer used? Can you remove them instead of commenting out?

Reply:
Sure I will remove. those are moved to the arch filelist, because bsg_dmc needs them.
Also, we cannot add modules twice to the list, otherwise the tools will complain.

Reply:
Got it. This makes sense!